src/components/AboutMeSection/AboutMeSection.js

```diff
@@ -52,10 +52,17 @@ const AboutMeSection = () => {
         )}
       </FormattedMessage>
       <SectionText>
-        <FormattedMessage id="description.aboutMeSection.partOne" />
-      </SectionText>
-      <SectionText>
-        <FormattedMessage id="description.aboutMeSection.partTwo" />
+        <FormattedMessage
+          id="description.aboutMeSection"
+          values={{
+            breakingLine: (
```

Comment:
i think that there are at least 2 line**s**. 
Anyway, wouldn't be cleaner to put make there two paragraphs with injected chunks, where the first element will have a margin ?